### ISSUE TITLE
midi is read-only (typescript error?)

### ISSUE THREAD

**Issue by glend1** [jazz-soft/web-midi-test#3]
when i try to use the following i get an error in my ide `midi` is read-only.

````
WMT.midi = false;
````

I dont know why this is happening, but the code seems to complete anyway because midi is failing as expected. so I assume this is a problem with the typescript definition.

**Comment by glend1** [comment]
im just importing it.

````
import * as WMT from 'web-midi-test';
````

**Comment by raphaelsiz** [comment]
I had a different error ("MidiSrc is not a constructor") which was resolved when I changed my import statement from the above to this:
```
import WMT from 'web-midi-test'
```

I also tried changing it back to what's in the README and changing `WMT.midi`, and also got an error ("Object is not extensible"). When I changed the import statement to `import WMT` again, it worked as expected.

TL;DR: I think the import statement listed in the README for TypeScript doesn't work, at least in our two cases. The import statement I listed above worked for my different situation, and it also worked when I replicated OP's situation to the best of my ability. 

Might be worth a change to the README, but I wasn't using TypeScript so I don't know for sure.
